docs/sources/mimir/operators-guide/monitor-grafana-mimir/installing-dashboards-and-alerts.md

```diff
@@ -67,3 +67,95 @@ Given the exact setup really depends on a case-by-case basis, the following inst
      },
    }
    ```
+
+### Deploy mixin with terraform
+
+Based on the earlier step, you can use [Terraform](https://www.terraform.io/) to deploy the mixin in both Grafana and Grafana Mimir. To do so you can use both [grafana/grafana](https://registry.terraform.io/providers/grafana/grafana/latest), [ovh/mixtool](https://registry.terraform.io/providers/ovh/mixtool/latest) and [ovh/mimirtool](https://registry.terraform.io/providers/ovh/mimirtool/latest) providers.
+
+1. Create terraform `main.tf`
+
+   ```hcl
+   # Specify which providers to use
+   terraform {
+     required_version = ">= 1.3.5"
+     required_providers {
+       mixtool = {
+         source  = "ovh/mixtool"
+         version = "~> 0.1.1"
+       }
+       mimirtool = {
+         source  = "ovh/mimirtool"
+         version = "~> 0.1.1"
+       }
+       grafana = {
+         source  = "grafana/grafana"
+         version = "~> 1.32.0"
+       }
+     }
+   }
+
+   # Configure providers if needed
+   provider "grafana" {
+     url  = "http://localhost:9000"
+     auth = "admin:admin"
+   }
+
+   provider "mimirtool" {
+     address   = "http://localhost:9009"
+     tenant_id = "anonymous"
+   }
+
+   locals {
+     mixin_source   = "custom.libsonnet"
+     jsonnet_path   = "vendor"
+   }
+
+   # Build alerts
+   data "mixtool_alerts" "mimir" {
+     source       = local.mixin_source
+     jsonnet_path = [local.jsonnet_path]
+   }
+
+   # Build rules
+   data "mixtool_rules" "mimir" {
+     source       = local.mixin_source
+     jsonnet_path = [local.jsonnet_path]
+   }
+
+   # Build dashboards
+   data "mixtool_dashboards" "mimir" {
+     source       = local.mixin_source
+     jsonnet_path = [local.jsonnet_path]
+   }
+
+   # Deploy rules
+   resource "mimirtool_ruler_namespace" "rules" {
+     namespace   = "rules_community"
+     config_yaml = data.mixtool_rules.mimir.rules
+   }
+
+   # Deploy alerts
+   resource "mimirtool_ruler_namespace" "alerts" {
+     namespace   = "alerts_community"
+     config_yaml = data.mixtool_alerts.mimir.alerts
+   }
+
+   # Deploy dashboards
+   resource "grafana_dashboard" "mimir" {
+     for_each    = data.mixtool_dashboards.mimir.dashboards
+     config_json = each.value
+   }
+   ```
+
+2. Initialise terraform
+   ```bash
+   terraform init
+   ```
+3. Review changes that terraform would apply on your infrastructure
```

Comment:
```suggestion
3. Review the changes that Terraform would apply on your infrastructure:
```
Is this preposition "on" or "to" here?

Reply:
I believe I should have used _to your infrastructure_

Example: https://developer.hashicorp.com/terraform/tutorials/aws-get-started/infrastructure-as-code#collaborate

> This lets you manage changes to your infrastructure through version control, as you would with application code.